app/models/article.rb

```diff
@@ -6,14 +6,32 @@ class Article < ApplicationRecord
   include Searchable
   include UserSubscriptionSourceable
 
+  include PgSearch::Model
+  multisearchable against: %i[body_markdown title],
+                  # do we need `keywords_for_search` ?
+                  associated_against: {
+                    organization: :name,
+                    tags: %i[name keywords_for_search],
+                    user: %i[username name]
+                  },
```

Comment:
this we should probably move to `against` + `additional_attributes` as associations are the antithesis of FTS indexes AFAIK